overlord/snapstate/flags.go

```diff
@@ -90,6 +90,10 @@ type Flags struct {
 	// This may eventually be set for specific snaps mentioned in the model
 	// assertion for non-dangerous grade models too.
 	ApplySnapDevMode bool `json:"apply-snap-devmode,omitempty"`
+
+	// Transaction will be true if the snap is required to
```

Comment:
Transactional is set to true to request that the set of snaps is transactionally installed/updated jointly.